pypit/artrace.py

```diff
@@ -957,10 +957,9 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False):
     msgs.info("Synchronizing left and right slit traces")
     # Define the array of pixel values along the dispersion direction
     xv = plxbin[:, 0]
-    num = (lmax-lmin)/2
+    num = (lmax-lmin)//2  # Should be an int, right?
```

Comment:
Yes, you're right. This wasn't a problem before python 3 support (i.e. future division).

Now that this is fixed, we should remove the comment to avoid confusion later.